src/partials/integrations/analytics/google.html

```diff
@@ -42,6 +42,14 @@
 
     /* Register virtual event handlers */
     document.addEventListener("DOMContentLoaded", function() {
+      if (document.forms.search) {
```

Comment:
So I read that you verified this works, correct? If so, LGTM.

Reply:
Yep, this worked for me.